cgroup2/utils_test.go

```diff
@@ -64,3 +64,22 @@ func TestToResources(t *testing.T) {
 	v2resources2 := ToResources(&res2)
 	assert.Equal(t, CPUMax("max 10000"), v2resources2.CPU.Max)
 }
+
+func BenchmarkReaduint64(b *testing.B) {
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		_ = getStatFileContentUint64("/proc/self/loginuid")
+	}
+}
+
+func BenchmarkReadSingleFile(b *testing.B) {
+	b.ReportAllocs()
+
+	out := make(map[string]string)
+	for i := 0; i < b.N; i++ {
+		if err := readSingleFile("/proc/self/loginuid", "", out); err != nil {
```

Comment:
readSingleFile expects a file with key value pairs, so `/proc/self/loginuid` is not a good choice.

Reply:
It doesn't afaict, it just places the resulting value in the passed in map. The original logic was to try and parse the output as a uint, and if it failed just treat the output as a string. It's only called for pids.current and pids.max which are both either "max" or some unsigned integer

Reply:
Sorry, I mixed that up with readKVStatsFile. Yes, this is the right approach then.